sdk/cosmos/azure-cosmos-spark_3-0_2-12/README.md

```diff
@@ -1,61 +1,42 @@
 # Azure Cosmos DB OLTP Spark connector client library for Java
 
-**Azure Cosmos DB OLTP Spark connector ** provides Apache Spark support for Azure Cosmos DB using 
+**Azure Cosmos DB OLTP Spark connector preview** provides Apache Spark support for Azure Cosmos DB using 
 the [SQL API][sql_api_query].
 [Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows 
 developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
 
-## Getting started
-TBD
+**NOTE this is a Preview build.**
```

Comment:
Please add in the Note
- This build has not been load or performance tested yet - and at this point is not recommended to be used in production scenarios

Reply:
- If you have any feedback or ideas on how to improve your experience please let us know here: https://github.com/Azure/azure-sdk-for-java/issues/new

Reply:
Thanks. DONE